### PR TITLE
Add documentation about `formatNumber` function

### PR DESCRIPTION
**Pull request by brusherru**
The function was introduced in https://github.com/xodio/xod/pull/1544

### CODE REVIEW

File: docs/reference/node-cpp-api/README.md [hunk 1]
```diff
@@ -208,66 +161,132 @@ Creates a new iterator pointing to the first element of the list.
...
+Converts Number into character string equivalent and put into `str`. Analog of `dtostrf`, `sprintf` and `to_string`, but optimized for microcontrollers, handles NaN, Inf and overflown numbers correctly and works on any platform.
+
+Examples:
+
+<table class="ui single line table">
+  <thead>
+    <tr>
+      <th class="four wide right aligned">value</th>
+      <th class="three wide right aligned">prec</th>
+      <th class="one wide"></th>
```
Comment (evgenykochetkov) [inline docs/reference/node-cpp-api/README.md]: Why the empty column?

**Comment by nkrkv** [comment]
:x: :x: :x:

Turkey! :bowling:

**Comment by nkrkv** [comment]
To add to @knopki comment: bump the minor number, not the patch.

File: docs/reference/node-cpp-api/README.md (hunk 1 above)
Comment (brusherru) [reply docs/reference/node-cpp-api/README.md]: To add some space between `prec` values and strings... Otherwise it looks ugly 😬

**Comment by brusherru** [comment]
@nkrkv @knopki @evgenykochetkov fixed!
And added a sentence what it returns.
Check it out, please

**Pull request merged**